### PR TITLE
add interactive map to collections sandbox data subpages

### PR DESCRIPTION
**Pull request by zcernigoj**
- added links to css and script for leaflet
- created `interactive-map.hbs` with code for interactive map
- changed compiling of the markdown files to html so that the code from `interactive-map.hbs` for interactive map is added to the compiled html files

### CODE REVIEW

File: _build/geojson/PS_polygons.geojson
Comment (dthiex) [inline _build/geojson/PS_polygons.geojson:1]: @dorothyron Probably something for you to update. The geojson seems to have some strange polygons in the middle of the ocean without points. 

grafik

File: _build/partials/interactive-map.hbs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    }
+
+    function generatePopupText(feature){
+        const functionString = () => navigator.clipboard.writeText(JSON.stringify(feature.geometry))
+
+        return "<h4>title</h4>" +
+            "<div>" +
+            "Time range:" + feature.properties.from + " to " + feature.properties.to + "<br>" +
+            "Area size: " + feature.properties["area / sqkmm"] + " km<sup>2</sup><br>" +
+            '<button class="leaflet-copy-geojson">copy geojson</button>' +
```
Comment (dthiex) [inline _build/partials/interactive-map.hbs:20]:
```suggestion
            '<button class="leaflet-copy-geojson">Copy GeoJSON</button>' +
```

File: _build/partials/interactive-map.hbs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        return {geojsonPoints, geojsonPolygons };
+    }
+
+    function generatePopupText(feature){
+        const functionString = () => navigator.clipboard.writeText(JSON.stringify(feature.geometry))
+
+        return "<h4>title</h4>" +
+            "<div>" +
+            "Time range:" + feature.properties.from + " to " + feature.properties.to + "<br>" +
+            "Area size: " + feature.properties["area / sqkmm"] + " km<sup>2</sup><br>" +
```
Comment (dthiex) [inline _build/partials/interactive-map.hbs:19]: Can we add some more padding between button and text as in the designs? It currently looks quite cramped.

grafik

File: _build/partials/interactive-map.hbs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+        return {geojsonPoints, geojsonPolygons };
+    }
+
+    function generatePopupText(feature){
+        const functionString = () => navigator.clipboard.writeText(JSON.stringify(feature.geometry))
+
+        return "<h4>title</h4>" +
+            "<div>" +
+            "Time range:" + feature.properties.from + " to " + feature.properties.to + "<br>" +
```
Comment (dthiex) [inline _build/partials/interactive-map.hbs:18]:
```suggestion
            "Time range: " + feature.properties.from + " to " + feature.properties.to + "<br>" +
```

**Review by dthiex — CHANGES_REQUESTED** [review]
Few more things that are different than in the designs
- point and API colour (Planet blue instead of blue)
- copy icon within "Copy GeoJSON" button
- highlight of selected point pin

File: _build/geojson/PS_polygons.geojson
Comment (Dorothyrono) [reply _build/geojson/PS_polygons.geojson:1]: Right 
@maximlamare could you check this

Comment (zcernigoj) [reply _build/geojson/PS_polygons.geojson:1]: locally, I needed to switch the coordinates, will check if it works without switching on test deploy

File: _build/partials/interactive-map.hbs (hunk 3 above)
Comment (zcernigoj) [reply _build/partials/interactive-map.hbs]: thanks, done

File: _build/partials/interactive-map.hbs (hunk 2 above)
Comment (zcernigoj) [reply _build/partials/interactive-map.hbs:19]: thanks, done

File: _build/partials/interactive-map.hbs (hunk 1 above)
Comment (zcernigoj) [reply _build/partials/interactive-map.hbs]: thanks, done

Comment (zcernigoj) [reply _build/partials/interactive-map.hbs]: Will add the icon when I will fix the button onclick function in a separate MR

**Comment by zcernigoj** [comment]
This is needed as soon as possible, so merging.

fixes will be done in separate MR

**Pull request merged**